### PR TITLE
Add Let's Go VGC

### PR DESCRIPTION
**Pull request by Ninnix**
Let's GO VGC was born from the idea of the famous player **Francesco Pardini** **([AlexisVGC](https://twitter.com/Alexis_Vgc))** and the team of the Italian website **[Altopiano Blu](https://www.altopianoblu.it/)**. The central idea of this format is to create a light, fun-based on the latest pokemon games (LGPE) as close as possible to the true vgc.

**Rules:** 
- The teams are composed of six pokemon but we only choose four of them as in the real vgc
- Absence of Skills
- Absence of the instruments held
- Possibility of choosing the Mega Evolution (for Pokémon that have more than one like Charizard)
- Absence of EVs (replaced by candies, but candies will not come into play in this metagame so there will be no mention in this guide)
- Choice of Pokémon restricted only to Kanto's Pokédex (and Alola forms)
- Very reduced Movepool compared to previous games

To better balance this metagame we had to prevent the use of some Pokémon and some moves:

**POKEMON NOT ALLOWED:**
- Mewtwo
- Mew
- Meltan
- Melmetal
- Pikachu 
- Eevee 

**MOVES NOT ADMITTED**
- Minimize
- Double Team
- Smoke Screen
- Sand Attack
- Horn Drill
- Fissure
- Sheer Cold
- Guillotine

The reasons why the 6 Pokémon are banned is simple: Mewtwo is simply too centralizing and strong given its very high statistics as well as access to two different mega evolutions that would make it all very random, as for the other three the ban is due to not possible to get them with the game only (Mew requires a Poké Ball Plus, Meltan and Melmetal require Pokémon Go and about the starter because they are unique). For the moves instead we are inspired by the Smogon banlist, which prevents you from using moves by OHKO and the moves that vary accuracy and avoidance.

**RESOURCES:**

[Smogon Thread ](https://www.smogon.com/forums/threads/let%E2%80%99s-go-vgc.3645766/#post-8004203) (awaiting approval from the OM team)

[Original Thread](https://www.altopianoblu.it/2018/11/30/lets-go-vgc) (Not updated with respect to the elimination of the starter)

### CODE REVIEW

**Comment by xfix** [comment]
I find it interesting you that a format apparently inspired by VGC bans OHKO and evasion moves.

The forum link doesn't work for me.

File: config/formats.js [hunk 1]
```diff
@@ -886,6 +886,22 @@ let Formats = [
 		ruleset: ['Pokemon', 'Allow AVs', 'Endless Battle Clause', 'Team Preview', 'HP Percentage Mod', 'Cancel Mod'],
 		banlist: ['Illegal', 'Unreleased'],
 	},
+	{
+		name: "[Gen 7 Let's Go] Let's Go VGC",
+		threads: [
+			`&bullet; <a href="https://www.smogon.com/forums/threads/let%E2%80%99s-go-vgc.3645766/">Lets go VGC</a>`,
```
Comment (xfix) [inline config/formats.js]: The forum link doesn't work for me.

**Comment by TheImmortal** [comment]
Formats aren’t just added to PS just for the sake of it, especially user-created ones. I’ll respond in the thread.

**Comment by Zarel** [comment]
On one hand, I'm pretty impressed by the amount of work that went into this.

On the other hand, I'd recommend testing this on a side server, first. The main server definitely isn't the right place for formats not sponsored by Smogon or Nintendo.

**Comment by MattHCVGC** [comment]
A fantastic idea. a perfect format, it would be really nice to have it available

**Comment by Zarel** [comment]
It's also really misleading to call it "VGC" when it's not officially sponsored by Nintendo.

**Pull request closed without merging**